modules/frontend/traceid_handlers.go

```diff
@@ -43,7 +43,7 @@ func newTraceIDHandler(cfg Config, next pipeline.AsyncRoundTripper[combiner.Pipe
 		}
 
 		// validate start and end parameter
-		_, _, _, _, _, reqErr := api.ValidateAndSanitizeRequest(req)
+		_, _, _, _, _, _, reqErr := api.ValidateAndSanitizeRequest(req)
```

Comment:
This function signature is getting lengthy.  Do we want to introduce a TraceByIDRequest object and use it as the output from `api.ParseTraceID(req)` ?

Reply:
Yes, I thought of that as well. What didn't convince me was that it also contains extra params `start` and `end` that are not part of `TraceByIDRequest` and it requires extra changes.

I don't have a strong opinion, but initially favored introducing only the necessary changes.